lib/src/json_rpc/methods.rs

```diff
@@ -1062,15 +1062,25 @@ pub enum SystemPeerRole {
 
 #[derive(Debug, Clone, serde::Serialize, serde::Deserialize)]
 pub enum TransactionStatus {
```

Comment:
nit: `#[serde(rename_all = "camelCase")]` could also work here 🙏

Reply:
I'm aware of this, but I don't like the magic of serde deciding of the names.